Remotion/Web/Core/UI/Controls/Command.cs

```diff
@@ -445,6 +445,20 @@ public virtual void RenderBegin (
 
       style.AddAttributesToRender(writer);
 
+      // We also check for commandInfo.OnClick as to ensure that we are actually rendering a clickable command.
+      // If we don't have access we would get commandInfo.onClick == null
```

Comment:
```suggestion
      // We also check for commandInfo.OnClick as to ensure that we are actually rendering a clickable command.
      // If we have a disable dcommand, we would get commandInfo.onClick == null. 
      // We accept that a disabled command isn't identified as a button to the screenreader for the time being.
      // This might be a future refactoring / improvement.
```